src/fitnesse/slim/StatementExecutor.java

```diff
@@ -131,7 +140,52 @@ private MethodExecutionResult getMethodExecutionResult(String instanceName, Stri
     }
     return results.getFirstResult();
   }
+ 
+  /**
+   *  
+   * @return true is the fixture will handles symbols assignments and lookups itself.
+   *         This should be a rare exception and is not recommended
+   *         
+   *  Would be nice to use the classname but we don't know it at this point.
+   *         
+   */
+  
+  private Boolean ignoreSymbols(String instanceName, String methodName){
+	  try{
```

Comment:
Hi Hans Joachim,

Thanks for your valid comments.
Swallowing all exceptions without any message is not best practice.

I want these methods to never fail!

Just as a background:
I don't recommend anybody to use the flag  SLIM_AGENT_FIXTURE_HANDLES_SYMBOLS
and these methods. The better solution is a different table type for your fixtures which is processed line by line. 
I added this as I feared to break   https://github.com/smartrics/RestFixture. Their code is currently only working because of a bug in the symbol handling. This PR fixes the bug. I wanted to give them a simple path to continue to work without changing all their code.
In the mean time I expect they even don't need it as they use a different symbol pattern in their fixture.

But as long it is not merged and I get feedback I can be only 99.99% sure.

In case I am wrong and there is a wider use for this the exception handling can be improved. 

In case I am right it is rather something to get removed in the next release. It should just be a backward compatibility option.

I can also remove it right away.
It comes to the general question how to handle possible incompatibilities.